README.md

```diff
@@ -18,3 +18,25 @@ This integration has been tested with Solarfocus eco<sup>manager-touch</sup> ver
 | Biomassboiler (_Kessel_) | :white_check_mark: | 
 
 _Note: The number of supported Heating Circuits, Buffers, and Boilers could be extended in the future_
+
+## Usage
+
+```python
+from pymodbus.client.sync import ModbusTcpClient as ModbusClient
+from pysolarfocus import SolarfocusAPI
+from pysolarfocus.const import PORT,Systems
+
+# Create a Modbus client
+client = ModbusClient(IP, port=PORT)
+client.connect()
+
+# Create the Solarfocus API client
+api = SolarfocusAPI(client, Systems.Vampair)
```

Comment:
```suggestion
solarfocus = SolarfocusAPI(client, Systems.Vampair)
```

Reply:
We probably have to rewrite the example code in the readme again when we change where the `Systems` enum is located.